### DISCUSSION TITLE
onboading issue: airflow users create does not work

### DISCUSSION THREAD

**Discussion by atomic** [apache/airflow#44901] [General]
### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
My answer to issue "something does not work" is "something is wrong". 

But what this something is and how your problem can be solved is literally impossible if you won't explain how things do not work, what you see and what your problem is in detail (including environment in which it happend).

Hardly recommend if you want to have a chance that someone will help you. (also this should be at most discussion - converting it to discussion if you want to continue and actually describe your issue).
